test/jdk/java/awt/ScrollPane/ScrollPaneLimitation.java

```diff
@@ -70,18 +72,12 @@ public void init() throws Exception {
     }
 
     public void start() throws Exception {
-
         // Check for Win32 environment. This test is only for Win32.
         // Even more: not for Windows 98: it doesn't support scroll children that long.
         try {
             Properties prop = System.getProperties();
-
             String os = prop.getProperty("os.name", "");
-            if (os != null &&
-                    (os.indexOf("98") == -1) &&
-                    (os.startsWith("Win") || os.startsWith("win") ||
-                            os.startsWith("WIN"))) {
-
+            if (os != null && (os.indexOf("98") == -1)) {
```

Comment:
Shall we drop the filter for Windows 98? Java cannot be run on Windows 9x for quite a long time, so it doesn't make sense to preserve this condition.

Reply:
Yeah, we can drop it if its not supported.